src/EssentialsPE/EventHandlers/SignEvents.php

```diff
@@ -164,7 +165,7 @@ public function onSignTap(PlayerInteractEvent $event): void{
                                 $item->setDamage(0);
                             }
                         }
-                        foreach ($event->getPlayer()->getInventory()->getArmorContents() as $item){
+                        foreach ($event->getPlayer()->getArmorInventory()->getContents() as $item){
                             if($this->getAPI()->isRepairable($item)){
                                 $item->setDamage(0);
```

Comment:
this won't work... it's modifying a copy of the object, not the object in the inventory itself. I know this is not a bug you introduced, but be aware that this won't work as intended anyway.